web/api/web_api_v1.c

```diff
@@ -776,16 +776,46 @@ static inline void web_client_api_request_v1_info_summary_alarm_statuses(RRDHOST
 static inline void web_client_api_request_v1_info_mirrored_hosts(BUFFER *wb) {
     RRDHOST *rc;
     int count = 0;
+
+    buffer_strcat(wb, "\t\"mirrored_hosts\": [\n");
     rrd_rdlock();
     rrdhost_foreach_read(rc) {
         if (rrdhost_flag_check(rc, RRDHOST_FLAG_ARCHIVED))
             continue;
-        if(count > 0) buffer_strcat(wb, ",\n");
+        if (count > 0)
+            buffer_strcat(wb, ",\n");
+
         buffer_sprintf(wb, "\t\t\"%s\"", rc->hostname);
         count++;
     }
-    buffer_strcat(wb, "\n");
+
+    buffer_strcat(wb, "\n\t],\n\t\"mirrored_hosts_status\": [\n");
+    count = 0;
+    rrdhost_foreach_read(rc)
+    {
+        if (rrdhost_flag_check(rc, RRDHOST_FLAG_ARCHIVED))
```

Comment:
Can you check with product if this is correct? Are they expecting child node databases to be available after a restart on the parent node?

Reply:
This is unchanged in current PR archived hosts were skipped before as well.
In case we need archived hosts in info payload we will need to update also the metadatalog to keep and comapct the claim_id. That however would not be 1hr task

Reply:
It was specifically asked to not expose archived host by Chris.